manifests/d/Diskuv/opam/2.2.0-alpha0-20221104/Diskuv.opam.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Diskuv.opam
+PackageVersion: 2.2.0-alpha0-20221104 # BUMP
+PackageLocale: en-US
+Publisher: Diskuv, Inc.
+PublisherUrl: https://www.diskuv.com/
+PrivacyUrl: https://diskuv.com/legal#privacy_policy
+PackageName: opam
+PackageUrl: https://opam.ocaml.org/
+License: LGPL-2.1-only WITH OCaml-LGPL-linking-exception
+LicenseUrl: https://github.com/ocaml/opam/blob/master/LICENSE
+Copyright: |
```

Comment:
The copyright should only really be one line.

Reply:
Changed it. Thanks!